src/components/map/components/action/layer-action/modify-action/index.js

```diff
@@ -1,36 +1,74 @@
 import { define } from "../../../../../../utils/core";
 import { VlMapLayerAction } from "../../layer-action";
 import { VlModifyAction } from "vl-mapactions/dist/vl-mapactions.js";
```

Comment:
`vl-mapactions/dist/vl-mapactions.js` import should occur before import of `../../../../../../utils/core`